notes/style-guide.md

```diff
@@ -50,7 +63,7 @@ This is very much a work-in-progress and is not exhaustive.
   ```
 
 * As can be seen in the example above, function arrows at line breaks
-  should  always go at the end of the line rather the beginning of the
+  should  always go at the end of the line rather than the beginning of the
```

Comment:
You didn't introduce it, but there's a double space here between "should" and "always" that should probably be fixed while we're doing this :smile:

Reply:
Yes!